se_denseslam/CMakeLists.txt

```diff
@@ -52,3 +52,19 @@ target_compile_definitions(${appname}-sdf PUBLIC ${field_type})
 list(APPEND BUILT_LIBS ${appname}-sdf)
 
 set(BUILT_LIBS ${BUILT_LIBS} PARENT_SCOPE)
+
+
+# --------------- INSTALL -----------------------
+INSTALL(
+        TARGETS se-denseslam-ofusion se-denseslam-sdf
```

Comment:
`TARGETS se-denseslam-ofusion se-denseslam-sdf` could be replaced with `TARGETS ${BUILT_LIBS}`. BUILT_LIBS contains all the map backends that were compiled so there is no need to enumerate them manually.